src/NLPModelsIpopt.jl

```diff
@@ -113,6 +115,9 @@ function ipopt(nlp :: AbstractNLPModel;
     end
   end
 
+  close(ipopt_log_file)
```

Comment:
`ipopt_log_file` is just a string with a file name. IPOPT opens and closes the stream.

Reply:
You're right. But something is happening inside the other solver I'm working on that calls IPOPT hundreds of times and the file is not being closed. I'm getting
```
SystemError: opening file "/tmp/jl_******": too many open files
```
It's hard to recreate the problem though, if `GC.gc()` is called, it looks like it closes everything.